packages/config/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@standardnotes/config",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

```suggestion
  "version": "2.0.0",
```
This would break consumer projects which have incompatible code so I would suggest incrementing the major version number to 2

---

consumer projects should be locked on fixed version to avoid situations like this. `2.0.0` is a major change - more of a project refactor kind of thing

---

so a question of semver interpretation: does this qualify as a major change?
semver says "increment the MAJOR version when you make incompatible API changes"
no API here, so do we treat a breaking change as an incompatible API change?
or do we differentiate (as semver seems to) between major breaking changes and non-major and only increment on major?
if so -- which is this one?
I don't know, it's all the same to me, but I prefer to stay on the safe side, so I am willing to bump it up to 2.0.0.
@karolsojko ok/nok?

---

If you update your dependency to another minor version, I personally wouldn't expect my build to stop working, which is what will happen with this change, hence my suggestion to increment the major version number. 
Though I also understand this is not much of a concern since it can easily be fixed :)

---

well, we're installing with the lockfile in mind so updating a version can cause breaks. The specifics of `config` package is a bit different to be honest - every change is something that you need to adjust in your code to be honest - but then again you're upgrading the package intentionally.

---

Having that said I'm against `2.0.0` since we would very quickly reach very odd package version like `54.0.0`

---

As I said, I don't have a strong opinion on this, but @baptiste-grob's heuristic (if the build in dependents will break, increment the major version) does sound convincing.

On the other hand, the build potentially breaks when adding any rule, if a dependent doesn't already follow it, so as @karolsojko points out, we would be incrementing the major version quite often.

High major versions might be fine though.

All in all, this doesn't seem very important, so to avoid prolonged bikeshedding I will leave the decision to the first of:

1. Whoever has overruling privilege here -- @karolsojko?, @mobitar?. Please comment if you would like to overrule.

2. Popular vote: :+1:  this post if you think the version should be '2.0.0', :-1: this post if you think it should be '1.4.0'.

Will merge tomorrow.

---

I'm giving the choice to vox populi as this is something we all will use :)

---

No strong opposition to 2.0, wins by votes, no PR update required, so merging. ;)
The debate can always continue at the next opportunity.